.github/workflows/deploy.yml

```diff
@@ -60,6 +56,20 @@ jobs:
       - name: Print packages
         run: npm list --depth=0
 
+      # This keeps package-lock.json up to date with the deployed game
+      - name: Commit the word list update if there are changes to commit
+        run: |-
+          if git diff --exit-code > /dev/null; then
+            echo "No changes to commit"
+          else
+            git add -A
```

Comment:
I could imagine this adding unexpected files someday, if something else in the workflow changed.

Reply:
ooh good point. Thanks! Fixed in https://github.com/skedwards88/crossjig/pull/28/commits/261455dc6d2e63fb73e01a8331e709c943167081 .